pop/README.md

```diff
@@ -0,0 +1,119 @@
+# Proof of Personhood
+
+pop command is used by one of the two groups of users:
+
+- organizers (org)
+- attendees (attendee)
+
+Each group has its own set of commands to start, finalize and use the keys.
+
```

Comment:
Regarding the rest of the section I would probably add a little more description such as: `Each organizers must run their own conode [link-to-conode] in order to setup a party`